tests/L0/run_transformer/test_fused_softmax.py

```diff
@@ -346,6 +346,8 @@ def test_forward(self, allmasked: bool=False):
 
     def test_backward(self, allmasked: bool=False):
         import generic_scaled_masked_softmax_cuda
+        prev_thresh = self.thresh
+        self.thresh = {"atol": 1.5e-1, "rtol": 5e-3}
```

Comment:
Are theses thresholds to be the same on all GPUs and dtypes?

Reply:
Yes, see line 319